### ISSUE TITLE
Rules Tab does not update with existing rules.

### ISSUE THREAD

**Issue by mister-yolh** [evilsocket/opensnitch#952]
**Describe the bug**
The Rules tab does not fill with the rules.

Include the following information:
 - OpenSnitch version: 1.6.0rc5
 - OS: KDE Neon
 - Version 5.27
 - Window Manager: KDE
 - Kernel version: Linux PC 5.15.0-69-generic 76-Ubuntu SMP Fri Mar 17 17:19:29 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

**To Reproduce**
Open GUI, select Rules tab, Application rules, Permanent/temporary. It's empty.

**Post error logs:** 
After launching opensnitch:
```
Themes not available. Install qt-material if you want to change GUI's appearance: pip3 install qt-material.
         ~ OpenSnitch GUI - 1.6.0rc5 ~
        protobuf: 3.12.4 - grpc: 1.51.1
-------------------------------------------------- 

Loading translations: /usr/lib/python3/dist-packages/opensnitch/i18n locale: pt_BR
Using server address: unix:///tmp/osui.sock
schema version: 2
db schema is up to date
using IPASN DB: /usr/lib/python3/dist-packages/data/ipasn_20140513_v12.dat.gz
exception loading ipasn db: [Errno 2] No such file or directory: '/usr/lib/python3/dist-packages/data/ipasn_20140513_v12.dat.gz'
Install python3-pyasn to display IP's network name.
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
_insert() ERROR INSERT OR REPLACE INTO rules (time, node, name, description, enabled, precedence, nolog, action, duration, operator_type, operator_sensitive, operator_operand, operator_data) VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?)
O número de parâmetros não corresponde
_insert() ERROR INSERT OR REPLACE INTO rules (time, node, name, description, enabled, precedence, nolog, action, duration, operator_type, operator_sensitive, operator_operand, operator_data) VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?)
new node connected, listening for client responses... /local
O número de parâmetros não corresponde
_insert() ERROR INSERT OR REPLACE INTO rules (time, node, name, description, enabled, precedence, nolog, action, duration, operator_type, operator_sensitive, operator_operand, operator_data) VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?)
O número de parâmetros não corresponde
```

When selecting the tabs:

```
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
setQuery() error:  Nenhuma consulta Não foi possível obter a linha
```

All entries under "System Rules", however, work as expected, and produce no terminal log. 

**Expected behavior (optional)**
The rules should appear on the window.

**Screenshots**
![image](https://github.com/evilsocket/opensnitch/assets/121654174/6b9d97c4-6828-4421-95e0-6ed520da485c)

**Comment by mister-yolh** [comment]
Hello @gustavo-iniguez-goya 
Yes, I upgraded from 1.5.2, although the gui was broken for me at the time, and I wasn't using it until I updated to 1.6.
I did not change anything in the database tab.
The events in the Events tab show up just fine.

Based on your post, I deleted the database on the cache folder, and that fixed it. Thanks.

**Comment by gustavo-iniguez-goya** [comment]
Thank you very much for the feedback!

I'll take it into account before the next release, to see if I can reproduce it.

**Comment by Sadi58** [comment]
@gustavo-iniguez-goya
I thought you might like to know:
I've experienced the same problem after upgrading from a working version 1.5.8.1-1+b2 from Debian Testing repositories, and it was fixed by deleting the file "~/.local/share/OpenSnitch_database".

However, there was still a minor problem with the number of rows to be displayed (set via the drop-down list at the bottom left). It was solved by changing the value there back and forth.

In fact, this minor element seems to be a bit problematic. Sometimes, it is possible to see several rows below the set limit (e.g. 50), starting from an arbitrary number like 14. I don't know what the number shown preceding that box means but it's sometimes the same as the set number, sometimes different.
